vcr/holder/wallet.go

```diff
@@ -90,6 +93,48 @@ func (h wallet) BuildPresentation(ctx context.Context, credentials []vc.Verifiab
 		}
 	}
 
+	switch options.Format {
+	case JWTPresentationFormat:
+		return h.buildJWTPresentation(ctx, *signerDID, credentials, options, key)
+	case "":
+		fallthrough
+	case JSONLDPresentationFormat:
+		return h.buildJSONLDPresentation(ctx, credentials, options, key)
+	default:
+		return nil, errors.New("unsupported presentation proof format")
+	}
+}
+
+// buildJWTPresentation builds a JWT presentation according to https://www.w3.org/TR/vc-data-model/#json-web-token
+func (h wallet) buildJWTPresentation(ctx context.Context, subjectDID did.DID, credentials []vc.VerifiableCredential, options PresentationOptions, key crypto.Key) (*vc.VerifiablePresentation, error) {
+	headers := map[string]interface{}{
+		jws.TypeKey: "JWT",
```

Comment:
check question: does the lib set the `kid`?

Reply:
yes

Reply:
yes